skia/BUILD.gn

```diff
@@ -368,6 +368,30 @@ component("skia") {
     "//third_party/skia/src/fonts/SkGScalerContext.h",
   ]
 
+  if (is_ios || is_mac || is_android) {
+    sources -= [
+      "//third_party/skia/src/gpu/gl/GrGLCreateNativeInterface_none.cpp"
+    ]
+
+    set_sources_assignment_filter([])
+
+    if (is_ios) {
+      sources += [
```

Comment:
If the filter is working then we could just list all 3 of these in one unified sources block.